spec/ts-smoke/electron/main.ts

```diff
@@ -940,7 +940,7 @@ app.whenReady().then(() => {
 
   appIcon.displayBalloon({
     title: 'Hello World!',
-    content: 'This the the balloon content.',
+    content: 'This is the balloon content.',
```

Comment:
This was a manual intervention -- my default would be to drop one of the `the`s...